open_instruct/grpo_fast.py

```diff
@@ -340,6 +342,8 @@ class Args:
     """whether to enable prefix caching"""
     vllm_top_p: float = 1.0
     """vLLM top p for nucleus sampling"""
+    inference_batch_size: Optional[int] = None
+    """Number of inference requests to batch together for vLLM processing"""
```

Comment:
maybe we could be more specific: (something like) "number of unique prompts sent to a single vllm engine at once"?